packages/fxa-settings/src/components/Settings/FlowSetup2faComplete/index.tsx

```diff
@@ -0,0 +1,138 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useEffect } from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import FlowContainer from '../FlowContainer';
+import GleanMetrics from '../../../lib/glean';
+import { useFtlMsgResolver } from '../../../models';
+import Banner from '../../Banner';
+import {
+  BackupCodesIcon,
+  BackupRecoverySmsIcon,
+  CheckmarkGreenIcon,
+} from '../../Icons';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { GleanClickEventType2FA } from '../../../lib/types';
+
+type FlowSetup2faCompleteProps = (
+  | {
+      backupType: 'code';
+      numCodesRemaining: number;
+    }
+  | {
+      backupType: 'phone';
+      lastFourPhoneDigits: string;
+    }
+) & {
+  serviceName: string;
+  onContinue: () => void;
+  reason?: GleanClickEventType2FA;
+};
+
+export const FlowSetup2faComplete = (props: FlowSetup2faCompleteProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+  const {
+    backupType,
+    serviceName,
+    onContinue,
+    reason = GleanClickEventType2FA.inline,
+  } = props;
+
+  useEffect(() => {
+    GleanMetrics.accountPref.twoStepAuthCompleteView({
+      event: { reason },
+    });
+  }, [reason]);
+
+  const backupMethodIcon =
+    backupType === 'code' ? (
+      <BackupCodesIcon className="flex-none" />
+    ) : (
+      <BackupRecoverySmsIcon className="flex-none" />
+    );
+
+  const localizedBackupMethod = ftlMsgResolver.getMsg(
+    `flow-setup-2fa-inline-complete-backup-${backupType}`,
+    backupType === 'code' ? 'Backup authentication codes' : 'Recovery phone'
+  );
+
+  const localizedBackupMethodInfo =
+    backupType === 'code' ? (
+      <FtlMsg
+        id="flow-setup-2fa-inline-complete-backup-code-info"
+        vars={{
+          count: props.numCodesRemaining,
+        }}
+      >
+        <p className="text-sm">{`${props.numCodesRemaining} ${
+          props.numCodesRemaining === 1 ? 'code' : 'codes'
+        } remaining`}</p>
+      </FtlMsg>
+    ) : (
+      // Phone numbers should always be displayed left-to-right,
+      // *including* in rtl languages
+      <p dir="ltr" className="text-sm">
+        •••••• {props.lastFourPhoneDigits}
+      </p>
+    );
+
+  return (
+    <FlowContainer hideBackButton={true}>
+      <Banner
+        type="success"
+        content={{
+          localizedHeading: ftlMsgResolver.getMsg(
+            'flow-setup-2fa-inline-complete-success-banner',
+            'Two-step authentication enabled'
+          ),
+        }}
+      ></Banner>
+      <div className="bg-grey-10 rounded-lg flex p-6 gap-5 items-center">
+        {backupMethodIcon}
+        <div>
+          <p className="font-bold text-lg">{localizedBackupMethod}</p>
```

Comment:
This looks a tad heavier than the design. Recommend:
```suggestion
          <p className="font-semibold text-md">{localizedBackupMethod}</p>
```